content/en/about/contributing/style/_index.md

```diff
@@ -59,8 +60,336 @@ Configuration is provided in `.prettierrc`, options can be set as follows:
 --print-width 80 --tab-width 2 --prose-wrap always
 ```
 
-When a Pull Request is received, the proposed changes are checked using
-[various linters](https://github.com/EGI-Federation/documentation/tree/main/.github/workflows).
+When a contribution is received (via a pull request), the proposed changes are
+checked using [various linters](https://github.com/EGI-Federation/documentation/tree/main/.github/workflows).
+
+## General writing guidelines
+
+Follow these guidelines to ensure readability and consistency of the EGI
+documentation.
+
+{{% alert title="Tip" color="info" %}} Short and simple sentences are easier
+to read and understand.
+{{% /alert %}}
+
+### Use standard English
+
+Use standard United States (U.S.) English throughout all technical
+publications. When in doubt about the spelling of a word, consult the
```

Comment:
I would recommend more British English/UK than US English, it's usually more what we are trying to use/follow and is more European.

Reply:
I can say that, but then I would have to guess where to look. MW is a good choice, in any case.

Reply:
On a second thought, all technical documentation I know is using US English.

Reply:
It's because is from US companies ;)
We are a European organisation, with mainly European users, it makes more sense to use a European language.
We also have many colleagues speaking native British English that can help.